exampleSite/content/about.md

```diff
@@ -3,7 +3,7 @@ title: About Hugo
 date: 2014-04-09
 authorbox: false
 sidebar: false
-menu: main
+menu: [main, footer, side]
```

Comment:
Why `side`?

Reply:
side over sidemenu: because it is obvious from the context that it is a menu. 

all three menus for about: to illustrate the menus.